bin/kona-exex/src/main.rs

```diff
@@ -1,3 +1,9 @@
+#![doc = include_str!("../README.md")]
+#![doc(issue_tracker_base_url = "https://github.com/paradigmxyz/op-rs/issues/")]
+#![warn(missing_debug_implementations, missing_docs, rustdoc::all)]
+#![deny(unused_must_use, rust_2018_idioms)]
```

Comment:
I think we can remove these because they're configured in the rustfmt config at the root level- following how reth configures linting